src/transform/EditTransform.js

```diff
@@ -0,0 +1,34 @@
+import './EditTransform.css'
+
+const CLASS = {
+  CONTAINER: 'pagelib_edit_section_link_container',
+  LINK: 'pagelib_edit_section_link',
+  PROTECTION: { UNPROTECTED: '', PROTECTED: 'page-protected', FORBIDDEN: 'no-editing' }
+}
+
+const DATA_ATTRIBUTE = { SECTION_INDEX: 'data-id', ACTION: 'data-action' }
+const ACTION_EDIT_SECTION = 'edit_section'
+
+/**
+ * @param {!Document} document
+ * @param {!number} index The zero-based index of the section.
+ * @return {!HTMLSpanElement}
+ */
+const newEditSectionLink = (document, index) => {
+  const container = document.createElement('span')
```

Comment:
Thoughts on decomposing the span bits out of the link function? Specifically, adding a new `newEditSectionSpan` function and making it public, then it would call `newEditSectionLink` to set the span contents. In this scenario I'd make `newEditSectionLink` private (moving its export to a `test` object) and `newEditSectionSpan` public.